tailscale/resource_logstream_configuration_test.go

```diff
@@ -0,0 +1,99 @@
+package tailscale_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	tsclient "github.com/tailscale/tailscale-client-go/v2"
+)
+
+const testLogStreamConfiguration = `
+	resource "tailscale_logstream_configuration" "test_logstream_configuration" {
+		log_type         = "configuration"
+		destination_type = "panther"
+		url              = "https://example.com"
+		token            = "some-token"
+	}`
+
+const testLogstreamConfigurationUpdate = `
+	resource "tailscale_logstream_configuration" "test_logstream_configuration" {
+		log_type         = "network"
```

Comment:
If you change the log type, it seems like we should probably delete the old log streaming configuration remotely?

Reply:
Maybe what we want is [ForceNew](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#forcenew) on the `log_type` field?

Reply:
Good call! Will adjust